### ISSUE TITLE
How to Reduce Overfitting With Dropout Regularization in Keras

### ISSUE THREAD

**Issue by huhuhang** [huhuhang/ReadList#152]
**标题**：How to Reduce Overfitting With Dropout Regularization in Keras

**字数**：1951 字

**摘要**：Dropout regularization is a computationally cheap way to regularize a deep neural network. Dropout works by probabilistically removing, or “dropping out,” inputs to a layer, which may be input variables in the data sample or activations from a previous layer.

**链接**：https://machinelearningmastery.com/how-to-reduce-overfitting-with-dropout-regularization-in-keras/
